deps.bzl

```diff
@@ -729,8 +729,9 @@ def prysm_deps():
         name = "com_github_ferranbt_fastssz",
         importpath = "github.com/ferranbt/fastssz",
         nofuzz = True,
-        sum = "h1:9VDpsWq096+oGMDTT/SgBD/VgZYf4pTF+KTPmZ+OaKM=",
-        version = "v0.0.0-20210120143747-11b9eff30ea9",
+        replace = "github.com/atif-konasl/fastssz",
```

Comment:
What is the motivation of this change?
IMHO we should not change it to your repository. Push it to lukso namespace or rethink if we need this replacement at all